api/v1alpha1/flowcollector_types.go

```diff
@@ -76,6 +76,7 @@ type FlowCollectorSpec struct {
 
 	// exporters defines additional optional exporters for custom consumption or storage. This is an experimental feature. Currently, only KAFKA exporter is available.
 	// +optional
+	// +k8s:conversion-gen=false
```

Comment:
hmm interesting. @acmenezes It seems we didn't try to put this `conversion-gen=false` on both `v1alpha1` and `v1beta1` and that does the trick !

Thanks @msherif1234 :partying_face: